<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addJoint">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add joint</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableJoints">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Joint</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Theta(deg)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>a(m)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>d(m)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>alpha(rad)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deleteJoint">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete joint</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableResult">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>z</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="computeResult">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
